packages/nodes-base/nodes/Adalo/OperationDescription.ts

```diff
@@ -0,0 +1,200 @@
+import {
+	INodeProperties,
+} from 'n8n-workflow';
+
+export const operationFields: INodeProperties[] = [
+	/**
+	 * get
+	 */
+
+	{
+		displayName: 'Element ID',
```

Comment:
Element ID or Row ID?

Reply:
They use Element ID internally, but for consistency, changed to Row ID in #3102